src/AssetConverter.php

```diff
@@ -255,16 +255,10 @@ private function buildConverterOptions(string $srcExt, array $options): string
         $command = '';
         $commandOptions = '';
         $path = '';
-        $options[$srcExt] = $options[$srcExt] ?? [];
 
-        foreach ($options[$srcExt] as $key => $value) {
-            if ($key === 'command') {
-                $command .= $value . ' ';
-            }
-
-            if ($key === 'path') {
-                $path = $this->aliases->get($value);
-            }
+        if (isset($options[$srcExt])) {
+            $command .= $options[$srcExt]['command'] . ' ';
```

Comment:
I think you need to ensure that `command` and `path` are exists in `$options[$srcExt]`

Reply:
done.